config/build.js

```diff
@@ -38,13 +38,9 @@ const svgo = new SVGO();
 
 names.forEach(async ({ inputFileName, outputComponentFileName, functionName }) => {
   const dom = await JSDOM.fromFile(inputFileName);
-  const shapes = dom.window.document.querySelector("defs").innerHTML;
+  const shapes = dom.window.document.querySelector("svg").innerHTML;
```

Comment:
Not sure this is a good move. This is there for the structure that Sketch exports.

Reply:
It's alright, because @darkwindcz is preparing every svg icon to the same structure as it's now. Sketch always export very messy svg with lot of unusable things there. That's the reason why I changed this file.